src/ec/packages/ec-component-pagination/pagination.html.twig

```diff
@@ -0,0 +1,78 @@
+{% spaceless %}
+
+{#
+  Parameters:
+  - "label" (string) (default: ''): for screen reader
+  - "items" (array) (default: []):
+    - "current" (boolean) (default: false)
+    - "previous" (boolean) (default: false)
+    - "next" (boolean) (default: false)
+    - "label" (string) (default: ''): label used when the item is not a link
+    - "aria_label" (string) (default: '')
+    - "link" (object) (default: ''): object of type Link
+  - "extra_classes" (optional) (string) (default: '')
+  - "extra_attributes" (optional) (array) (default: [])
+    - "name" (string) Attribute name, eg. 'data-test'
+    - "value" (string) Attribute value, eg: 'data-test-1'
+#}
+
+{# Internal properties #}
+
+{% set _label = label|default('') %}
+{% set _items = items|default([]) %}
+{% set _language = language|default('') %}
+
+{% set _css_class = 'ecl-pagination' %}
+{% set _extra_attributes = '' %}
+
+{# Internal logic - Process properties #}
+
+{% if extra_classes is defined and extra_classes is not empty %}
+  {% set _css_class = _css_class ~ ' ' ~ extra_classes %}
+{% endif %}
+
+{% if extra_attributes is defined and extra_attributes is not empty and extra_attributes is iterable %}
+  {% for attr in extra_attributes %}
+    {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name ~ '="' ~ attr.value ~ '"' %}
+  {% endfor %}
+{% endif %}
+
+{# Print the result #}
+
+<nav class="{{ _css_class }}"{{ _extra_attributes|raw }}>
+  <ul class="ecl-pagination__list">
+    {% for _item in _items %}
+      <li class="ecl-pagination__item{{ _item.current ? ' ecl-pagination__item--current' }}{{ _item.previous ? ' ecl-pagination__item--previous' }}{{ _item.next ? ' ecl-pagination__item--next' }}">
```

Comment:
Couldn't we have a smarter way of handling this? I just mean that we have a correspondence between the keys in the "link item" (current, previous, next) and the name of the modifier classes, maybe we could set the base class and then add the modifier when the key is not empty instead of having to explicitely set them here.
Just an idea..